### DISCUSSION TITLE
Is there an opensource tool or SAAS platform that can quickly deploy a civitai model?

### DISCUSSION THREAD

**Discussion by Judy0406-CJ** [civitai/civitai#889] [Q&A]
opensource tool: something like https://github.com/huggingface/text-generation-inference, but for Stable diffusion model.
Saas platform: huggingface endpoint inference and replicate.com can host model, but need to transform to their specific format.

**Comment by lee101** [comment]
in my opinion [Netwrck/stable-diffusion-server](https://github.com/Netwrck/stable-diffusion-server) is designed to be this, or comfyui/webui i think might work to
